src/styles/blocks/stars.scss

```
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-image: url(../images/star.svg);
+    margin-right: 4px;
+
+    :last-child {
```

I think you are missing the `&` here
```suggestion
    &:last-child {
```